tensornetwork/contractors/opt_einsum_paths/utils.py

```diff
@@ -12,10 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Helper methods for `path_contractors`."""
-
+import tensornetwork as tn
```

Comment:
Same comment as before

For tests outside of `tests/`, prefer to just import the modules you actually need.

Reply:
And on that note, everything INSIDE of `tests/` should ONLY have `import tensornetwork as tn` so we can make sure the public API behaves as we expect it to behave.